plugins/topic.py

```diff
@@ -32,19 +32,15 @@ def make_topic(self, msg, new_topic=None, new_music=None):
             )
             self.bot.set_topic(topic_string, msg.chan)
 
-    async def find_title(self, url):
-        async with aiohttp.ClientSession() as session:
-            async with session.get(url) as r:
-                page_bytes = await r.read()
-
-        page = page_bytes.decode("utf-8")
-        title_start = page.find("<title>")
-        title_end = page.find("</title>")
-        if title_start > 0 and title_end > 0:
-            title_start += len("<title>")
-            return page[title_start:title_end]
-        else:
-            return ""
+    def find_title(self, url):
```

Comment:
Does this revamped `find_title()` method work for anything else than Youtube ? (think of Soundcloud, Vimeo, etc...)

Reply:
Yup. It even works with Pornhub (and github. I tried. And if it fails, it will return a bullshit string)